src/sock/km/sock.c

```diff
@@ -109,6 +111,38 @@ FnSockStreamSocketReceive(
     _Inout_ SIZE_T* BytesAccepted
     );
 
+_Function_class_(DRIVER_INITIALIZE)
+_IRQL_requires_same_
+_IRQL_requires_(PASSIVE_LEVEL)
+NTSTATUS
+DriverEntry(
```

Comment:
Since you don't have a driver service, I _think_ the DriverEntry isn't necessary, but since it doesn't hurt we can leave it.

Reply:
It is unused (not called), but the driver framework seems to expect it

```
    2>WdfDriverEntry.lib(stub.obj) : error LNK2019: unresolved external symbol DriverEntry referenced in function FxDr
       iverEntryWorker [N:\repos\win-net-test\src\sock\km\fnsock_km.vcxproj]
     2>N:\repos\win-net-test\artifacts\bin\amd64chk\fnsock_km.sys : fatal error LNK1120: 1 unresolved externals [N:\rep
       os\win-net-test\src\sock\km\fnsock_km.vcxproj]
```